pymc4/_random_variables.py

```diff
@@ -7,11 +7,28 @@
 from . import _template_contexts as contexts
 
 import sys
-import tensorflow_probability.distributions as tfd
-
+import numpy as np
+import tensorflow_probability as tfp
+from tensorflow_probability import distributions as tfd
+
+
+# Random variables that PyMC3 support, but tfp does not support as
```

Comment:
PyMC4? :)

Reply:
@fonnesbeck was confused when I originally wrote PyMC4! 😅

PyMC4 _currently_ doesn't have any of those random variables, but after this PR it will... so I think we should be saying PyMC4?

Reply:
Oh now I get it, it's fine then.

Reply:
*supports